easybuild/easyconfigs/m/METIS/METIS-5.1.0-intel-2014b-32bitIDX.eb

```diff
@@ -0,0 +1,23 @@
+name = 'METIS'
+version = '5.1.0'
+versionsuffix = '-32bitIDX'
```

Comment:
boegel: Well, the other special thing is the absence of the `METIS_IDXTYPEWIDTH.patch`
For some reason `OpenFOAM 2.3.0` needs 32 bit index, probably beacuse it uses fortran code. Instead of patching `OpenFOAM`, I made the 32bit index version of `METIS`.